composer.json

```diff
@@ -72,8 +89,6 @@
   "config": {
     "sort-packages": true,
     "preferred-install": {
-      "drupal/lb_ux": "source",
```

Comment:
we should leave these settings, to keep `--prefer-dist` options for `composer install` in Makefile

Reply:
Due to the new repo settings these lines weren't really needed cause we will always clone the repo so no `dist` available. Anyway it doesn't affect so we can leave like this and create a new PR to do cleanups.